### PR TITLE
Add 526 boolean for terminallyIll

### PR DESCRIPTION
**Pull request by hinzed1127**
Corresponding schema change for department-of-veterans-affairs/vets-website#9930

### CODE REVIEW

File: dist/21-526EZ-ALLCLAIMS-schema.json [hunk 1]
```diff
@@ -1016,6 +1016,9 @@
         }
       }
     },
+    "terminallyIll": {
```
Comment (SMLuthi) [inline dist/21-526EZ-ALLCLAIMS-schema.json]: The BE is looking for `isTerminallyIll` in the form.

File: src/schemas/21-526EZ-allclaims/schema.js [hunk 2]
```diff
@@ -488,6 +488,9 @@ const schema = {
         }
       }
     },
+    terminallyIll: {
```
Comment (SMLuthi) [inline src/schemas/21-526EZ-allclaims/schema.js]: Same here

File: dist/21-526EZ-ALLCLAIMS-schema.json (hunk 1 above)
Comment (hinzed1127) [reply dist/21-526EZ-ALLCLAIMS-schema.json]: sounds good, will update

Comment (SMLuthi) [reply dist/21-526EZ-ALLCLAIMS-schema.json]: Thanks!

**Pull request merged**